de.k_bo.Televido.json

```diff
@@ -58,8 +58,8 @@
       "sources": [
         {
           "type": "archive",
-          "url" : "https://github.com/d-k-bo/televido/releases/download/v0.1.0/televido-0.1.0.tar.xz",
-          "sha256": "8abaf53646d278d69f78f985e33398f4e3e081b4f41f713ed8ec777bba38198c"
+          "url": "https://github.com/d-k-bo/televido/releases/download/v0.1.0/televido-0.1.0.tar.xz",
+          "sha256": "025df8987dd420e2d5d923fe13426bf9dfa7a90c67ae08e8838c5da415bca4a1"
```

Comment:
retagging is bad.

Reply:
Sorry, I'm pretty new to packaging. I believed this would be the right thing to do since the app itself didn't change.

Reply:
It seems like I misunderstood the concept of git tags. I was not aware that they are meant to be immutable.